server/app.js

```diff
@@ -1,22 +1,59 @@
 const express = require('express');
-const bodyParser = require('body-parser');
+const parser = require('body-parser');
```

Comment:
You could remove body parser altogether. You are already using express.json which is the built-in parser that works the same. You could then remove a dependeny and reduce load times.

Reply:
Agreed! I usually use express for parsing but these were already there. I will remove them.
Thanks